Gemfile

```diff
@@ -56,3 +56,5 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+
+gem "byebug", "~> 11.1", groups: %i[development test]
```

Comment:
You only need this on development or test

Reply:
You're right. I'm leaving it only for dev.